include/mme-app/mme_app.h

```diff
@@ -31,6 +31,18 @@ extern "C"{
         integrity_order : [ EIA1, EIA2, EIA0 ]
         ciphering_order : [ EEA0, EEA1, EEA2 ]
 */
+
+typedef struct dns_config
+{
+        unsigned int dns_flag;
+	unsigned int concurrent;
+	unsigned int percentage;
+	unsigned int interval_seconds;
+	unsigned int query_timeout_ms;
+	unsigned int query_tries;
+        char* dns1_ip;
```

Comment:
I would like to avoid dns1_ip name. Can we not remove 1 ?

Reply:
This 1 was added purposly so that if we need to need multiple DNS support in future. Do I still need to remove it?